src/04-modality-specific-files/04-intracranial-electroencephalography.md

```diff
@@ -246,6 +256,18 @@ SHOULD be present:
 | status             | OPTIONAL              | Data quality observed on the channel (good/bad). A channel is considered bad if its data quality is compromised by excessive noise. Description of noise type SHOULD be provided in `[status_description]`.                                                                |
 | status_description | OPTIONAL              | Freeform text description of noise or artifact affecting data quality on the channel. It is meant to explain why the channel was declared bad in `[status]`.                                                                                                               |
 
+{{ MACROS___make_columns_table(
+   {
+      "reference": "OPTIONAL",
```

Comment:
this makes references to the EEG section here ... but we are in the iEEG section of the spec - so these references don't make sense here.

Reply:
The definitions for EEG and iEEG are _really_ different, so I decided to split it into two schema objects. I don't _love_ splitting definitions for the same column in the same file across modalities, so if there's any way to write a single definition that works for both that would be awesome. Do you think it's possible?